### PR TITLE
[improve][workflow] Upgrade to latest docbot action

### PR DESCRIPTION
**Pull request by maxsxu**
Master Issue: #15797 https://github.com/apache/pulsar-test-infra/issues/67

### Motivation

Simplify invocation of docbot action

### Modifications

Upgrade to latest docbot action

### Verifying this change

- [x] Make sure that the change passes the CI checks.

This change is a trivial rework / code cleanup without any test coverage.

### Documentation

Check the box below or label this PR directly.

Need to update docs? 

- [ ] `doc-required` 
(Your PR needs to update docs and you will update later)
  
- [ ] `doc-not-needed` 
(Please explain why)
  
- [x] `doc` 
(Your PR contains doc changes)

- [ ] `doc-complete`
(Docs have been already added)

### CODE REVIEW

**Comment by maxsxu** [comment]
We need to merge with this [PR](https://github.com/apache/pulsar-test-infra/pull/68) simultaneously.

/cc @tisonkun @Anonymitaet @lhotari

File: .github/workflows/ci-documentbot.yml [hunk 1]
```diff
@@ -38,19 +38,8 @@ jobs:
...
-          ref: master
-
-      - name: Set up Go
-        uses: actions/setup-go@v3
-        with:
-          go-version: 1.18
-
       - name: Labeling
-        uses: ./docbot
+        uses: pulsar/pulsar-test-infra/docbot@master
```
Comment (tisonkun) [inline .github/workflows/ci-documentbot.yml:42]:
```suggestion
        uses: apache/pulsar-test-infra/docbot@master
```

**Review by tisonkun — COMMENTED** [review]
I think we should merge this patch after https://github.com/apache/pulsar-test-infra/pull/68 merged?

File: .github/workflows/ci-documentbot.yml (hunk 1 above)
Comment (maxsxu) [reply .github/workflows/ci-documentbot.yml]: Fixed

**Review by urfreespace — APPROVED** [review]

**Review by Anonymitaet — APPROVED** [review]

**Comment by maxsxu** [comment]
We can merge this PR now as the https://github.com/apache/pulsar-test-infra/pull/68 is merged.

**Review by nodece — APPROVED** [review]

**Comment by maxsxu** [comment]
> /pulsarbot run-failure-checks

@Anonymitaet These failure checks introduced in master is irrelevant to this PR.

**Pull request merged**